public/data/OkeyChukwude.json

```
@@ -0,0 +1,23 @@
+{
+    "name": "Okechukwu Chukwude",
+    "type": "personal",
+    "bio": "Backend Engineer, Computer Vision Enthusiast, Learning Web3.0",
+    "avatar": "https://github.com/OkeyChukwude.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/OkeyChukwude",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/OkeyChukwude",
+        "icon": "twitter"
+      },
+      {
+        "name": "Connect with me on LinkedIn",
+        "url": "https://www.linkedin.com/in/okechukwu-chukwude-8a5b8020a",
+        "icon": "linkedin"
+      }
+    ]
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```